lib/sedna/common/config.py

```diff
@@ -269,9 +269,16 @@ class BaseConfig(ConfigSerializable):
     # the name of FederatedLearningJob and others Job
     job_name = os.getenv("JOB_NAME", "sedna")
 
+    pretrained_model_url = os.getenv("PRETRAINED_MODEL_URL", "./")
     model_url = os.getenv("MODEL_URL")
     model_name = os.getenv("MODEL_NAME")
 
+    transmitter = os.getenv("TRANSMITTER", "ws")
```

Comment:
These `transmitter`/`agg_data_path` config are federated-related, better to new another config class